style.css

```diff
@@ -88,6 +91,632 @@ a {
+  height: 200px;
+  width: 125px;
+  position: absolute;
+  top: 21.5%;
+  right: 0;
+}
+
+.about-categories {
+  padding-top: 5rem;
+  padding-bottom: 5rem;
+}
+
+.categories {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.categories-header-container {
+  display: flex;
+  align-items: center;
+  padding: 0.6em 1em 0 1em;
+  margin: 0;
+  gap: 1em;
+}
+
+.categories-header {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.6rem;
+  color: #fff;
+}
+
+.categories-list {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 1.2rem;
+  margin: 0;
+}
+
+.resume-btn {
+  font-family: 'Inter', sans-serif;
+  background: #ff6b00;
+  text-decoration: none;
+  padding: 0.8rem;
+  margin-top: 1.5rem;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  transition: all 0.7s;
+}
+
+.resume-btn:hover {
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+#contact-me h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  text-align: center;
+  color: #172b4d;
+  margin: 120px 24px 58px 24px;
+}
+
+#contact-form ul {
+  display: flex;
+  flex-direction: column;
+  border: none;
+  padding: 0 24px;
+}
+
+#contact-form ul li {
+  text-align: center;
+  margin-bottom: 24px;
+}
+
+input,
+textarea {
+  height: 40px;
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  width: 100%;
+  outline: 0;
+}
+
+textarea {
+  color: #3c3a39;
+  min-height: 120px;
+  background: #fbf8f7;
+  padding-top: 10px;
+}
+
+#contact-me button:hover {
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+#contact-me button:active {
+  color: #f4f1f0;
+  background: #e05304;
+}
+
+ul.footer-list {
+  display: flex;
+  justify-content: center;
+  padding: 0;
+}
+
+.footer-li {
+  margin: 0 11px;
+}
+
+@media screen and (min-width: 768px) {
+  .menu_items {
+    display: flex;
+    font-family: 'Inter', sans-serif;
+  }
+
+  nav ul li {
+    list-style-type: none;
+  }
+
+  nav ul li:nth-child(1) {
+    display: none;
+  }
+
+  nav ul li span {
+    display: none;
+  }
+
+  nav ul li a {
+    margin: 10px;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    width: 45px;
+    height: 20px;
+    color: #fff;
+  }
+
+  .menuImg {
+    display: none;
+  }
+
+  header {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 7% 0 7%;
+  }
+
+  #headline {
+    display: flex;
+    justify-content: start;
+    position: relative;
+    background-image: url('./images/Illustration-header.svg');
+    background-position: top -2em right 0;
+    background-size: 30em;
+    height: 57vh;
+  }
+
+  .headlines {
+    width: 566px;
+    padding-top: 16vh !important;
+  }
```

Comment:
- [ ] Your headline section is good and looks almost the same as the Figma template. But I recommend not to use `vh` to set paddings, margins, or height values because on different screen sizes it can have a not expected look, and sometimes other elements can overlap. On a smaller screen, the works section overlaps your headline section. Kindly change that.
 - Your design:
 - 
<img width="1242" alt="image" src="https://user-images.githubusercontent.com/61919617/206689555-c1b7b9d4-cca8-4617-947f-a089408aaa90.png">

Reply:
Understood

Reply:
you are right